docs/widgets/iframe/index.mdx

```diff
@@ -60,3 +60,12 @@ Check out our documentation on [editing tiles](/docs/getting-started/after-the-i
 
 ## Screenshots
 ![](./widget.webp)
+
+---
+
+## homarr-iframes
+The project [homarr-iframes](https://github.com/diogovalentte/homarr-iframes) has iframes designed to be used with Homarr. These iframes offer a minimalist view of other systems, like viewing your tasks from [Vikunja](https://github.com/go-vikunja/vikunja) or bookmarks from [Linkwarden](https://github.com/linkwarden/linkwarden).
+
+![](./homarr-iframes-example.webp)
```

Comment:
Add an alternative text to this image. Can be useful for blind people and screen readers.